google_drive/google_docs.py

```diff
@@ -121,10 +121,15 @@ def check_url(self, data, suffix=''):  # pylint: disable=unused-argument,no-self
         """
         Checks that the given document url is accessible, and therefore assumed to be valid
         """
-        test_url = data['url']
         try:
+            test_url = data['url']
             url_response = requests.head(test_url)
-        # Catch wide range of errors
+        except KeyError as ex:
```

Comment:
Nit, but I would put the two lines in two different try/except blocks, since you're catching different exceptions for each. This way you don't risk to inadvertently catch a `KeyError` that would be thrown for a different reason from `requests.head()` for example.